net/hclouddns/src/opnsense/scripts/ddclient/lib/account/hetzner_cloud.py

```diff
@@ -24,12 +24,16 @@
     POSSIBILITY OF SUCH DAMAGE.
 
     Hetzner Cloud DNS API provider for OPNsense DynDNS
-    Uses the new Cloud API (api.hetzner.cloud) instead of the deprecated dns.hetzner.com API
+    Uses the new Cloud API (api.hetzner.cloud) with proper rrset-actions endpoints
 """
 import syslog
+import time
 import requests
 from . import BaseAccount
 
+ACTION_POLL_INTERVAL = 0.5  # seconds between action status polls
```

Comment:
This is rather short, what about 1 second ? Ideally you would use an exponential backoff function.